### DISCUSSION TITLE
How can we get/access OmniPVD?

### DISCUSSION THREAD

**Discussion by JailbreakPapa** [NVIDIA-Omniverse/PhysX#15] [Q&A] [answered]
through the docs it states we can access it through an extension, but its not very clear exactly where its located (within NVIDIA Omniverse Create)

**Comment by JailbreakPapa** [comment, marked as answer]
Just answered my own question: its not located through the actual omniverse launcher, its located within the creates extension browser.


![image](https://user-images.githubusercontent.com/67095081/200876714-bdf035c7-8afa-488b-96de-0753873547d7.png)

**Comment by amoravanszky** [comment]
Please give feedback on OmniPVD.  I am happy to hear any feedback about your experience with it.  It is still early days for this tool but we are very actively developing it.

**Reply by MunoDevelop** [reply]
what i thought was real-time 3d visualizer so newbee dev like me "not" need to draw all basic things myself. but its looks not like that

**Comment by MunoDevelop** [comment]
Any developers: if what you need is runtime visual debugger , download from : https://developer.nvidia.com/tools-downloads  , search with PhysXVisualDebugger then you will have this: 
![image](https://github.com/user-attachments/assets/9c2dff05-c0c3-4947-9be3-e10af84781e4)
runtime it will be like this: 
![image](https://github.com/user-attachments/assets/130bffca-1836-423e-919e-960d89e42869)
